app/views/users/index.html.erb

```diff
@@ -1 +1,13 @@
-<h1>list of users is here</h1>
\ No newline at end of file
+<% @users.each do |user| %>
```

Comment:
### Optional
- [ ] It will be a great idea to use partials to render some part of our codebase

Reply:
Yes I will surely implement that in the next milestones! Thanks much appreciated